discord/role.py

```diff
@@ -66,7 +66,7 @@ class Role(DiscordObject):
     _mentionable: bool
     _tags: Optional[RoleTags]
 
-    def __init__(self, guild: Guild, cache: RoleCache, payload: RolePayload):
+    def __init__(self, guild: Guild, cache: LFUCache, payload: RolePayload):
         self._guild = guild
         self._cache = cache
```

Comment:
Cache shouldn't be stored in the models. If this could be removed and all the mentions of it?

Reply:
`cache` was already a parameter of `Role.__init__` so I assumed that it'll be like that and changed the typehint since the `RoleCache` class was removed. I'll remove the parameter entirely

Reply:
> Cache shouldn't be stored in the models. If this could be removed and all the mentions of it?

How do we pass the cache tho?